cmd/atlas/internal/cmdapi/project_test.go

```diff
@@ -27,6 +27,7 @@ env "local" {
 	dev = "docker://mysql/8"
 	src = "./app.hcl"
 	schemas = ["hello", "world"]
+	revisions_schema = "revisions"
```

Comment:
Looks good, but I'd love to merge it with `migration` in next PRs:
```hcl
migration {
    url = "file://migrations"
    revisions_schema = "revisions"
}
```

Reply:
cc @rotemtam

Reply:
what do you mean by that?